buildroot-external/rootfs-overlay/usr/libexec/hassos-apparmor

```diff
@@ -1,12 +1,18 @@
 #!/bin/sh
 set -e
 
+APPARMOR_URL="https://version.home-assistant.io/apparmor.txt"
 PROFILES_DIR="/mnt/data/supervisor/apparmor"
 CACHE_DIR="${PROFILES_DIR}/cache"
 REMOVE_DIR="${PROFILES_DIR}/remove"
 
 # Check folder structure
-mkdir -p "${PROFILES_DIR}"
+if [ ! -d "${PROFILES_DIR}" ]; then
+    echo "[INFO]: AppArmor profile missing, downloading..."
+    mkdir -p "${PROFILES_DIR}"
+    systemctl start network-online.target
```

Comment:
we can also add network.target to wanted on the service file

Reply:
That would delay AppArmor on every boot. With this method, we only do it if the profile is missing. Since that is almost never the case, I prefer to have that dependency not by default.